route.go

```diff
@@ -43,10 +49,11 @@ type Route interface {
 }
 
 type RouteSnapshot struct {
-	Matcher Matcher        `json:"matcher"`
-	Dests   []*Destination `json:"destination"`
-	Type    string         `json:"type"`
-	Key     string         `json:"key"`
+	Matcher   Matcher        				`json:"matcher"`
+	Dests     []*Destination 				`json:"destination"`
+	Type      string         				`json:"type"`
+	Key       string         				`json:"key"`
+	Replicas  map[string][]*Destination    	`json:"replicas"`
```

Comment:
we can move `json:"replicas"` to more right to align with other fields (for consistency sake).